android/src/ti/map/AnnotationProxy.java

```diff
@@ -351,34 +363,23 @@ public void onPropertyChanged(String name, Object value)
 			updateInfoWindow();
 		} else if (name.equals(TiC.PROPERTY_LEFT_BUTTON)) {
 			getOrCreateMapInfoWindow().setLeftOrRightPane(value, TiMapInfoWindow.LEFT_PANE);
-			if (value == null) {
-				Object leftView = getProperty(TiC.PROPERTY_LEFT_VIEW);
-				if (leftView != null) {
-					getOrCreateMapInfoWindow().setLeftOrRightPane(leftView, TiMapInfoWindow.LEFT_PANE);
-				}
-			}
```

Comment:
Please explain me again, why this removal is ok. From what I see, it handles a null-check for `value` and supplies the `leftView` property for that case instead (which makes sense and would be breaking change to be removed). Same for `rightView`.